libs/core/langchain_core/utils/_merge.py

```diff
@@ -23,6 +24,9 @@ def merge_dicts(left: Dict[str, Any], right: Dict[str, Any]) -> Dict[str, Any]:
             merged[k] = v
         elif v is None or merged[k] == v:
             continue
+        # TODO Numeric type data is temporarily passed.
+        elif isinstance(v, numbers.Number):
+            continue
```

Comment:
not sure this generalizes

when/why would you get two numbers and want to ignore one?

Reply:
For example, the timestamp attribute should not be merged.

Reply:
> not sure this generalizes
> 
> when/why would you get two numbers and want to ignore one?

So skip it here instead of throwing an exception directly? 


This is the real data when I use qianfan, They encounter a timestamp merge exception when merging.

![image](https://github.com/langchain-ai/langchain/assets/48236177/c81b03b7-3d91-4a17-a7ed-d77a3e73ddbb)


```
{'id': 'as-juqwpqiphi', 'object': 'chat.completion', 'created': 1709774979, 'sentence_id': 0, 'is_end': False, 'is_truncated': False, 'result': 'Of course', 'need_clear_history': False, 'finish_reason': 'normal', 'usage': {'prompt_tokens': 5, 'completion_tokens': 0, 'total_tokens': 5}}
```

```
{'id': 'as-juqwpqiphi', 'object': 'chat.completion', 'created': 1709774980, 'sentence_id': 1, 'is_end': False, 'is_truncated': False, 'result': ', I\'d be happy to tell you a story. Here\'s a classic fairy tale called "The Ugly Duckling":\n\nOnce upon a', 'need_clear_history': False, 'finish_reason': 'normal', 'usage': {'prompt_tokens': 5, 'completion_tokens': 0, 'total_tokens': 5}}
```


There needs to be a better merging strategy to support

Reply:
something like timestamp should probably be dropped from streaming chunks before it comes to merging them (or timestamp field should be converted to a list that's appended to)

Reply:
> something like timestamp should probably be dropped from streaming chunks before it comes to merging them (or timestamp field should be converted to a list that's appended to)

Do you mean not to change merge method but to format the chunk in Qianfan?